datacube_alchemist/cli.py

```diff
@@ -4,13 +4,16 @@
 import click
 from datacube.ui import click as ui
 
-from datacube_alchemist._utils import _get_logger
+from datacube_alchemist._utils import _configure_logger
+import structlog
 
 from odc.aws.queue import get_queue
 
 from datacube_alchemist.worker import Alchemist, get_messages
 
-_LOG = _get_logger("CLI")
+_configure_logger()
```

Comment:
Although it usually doesn't matter, it's better to put this inside `main()` or `cli()` so that it executes when run as a program, but not when the module is imported.

It's confusing and hard to debug when importing a module causes an action, like changing the global logging configuration.